package.json

```diff
@@ -31,7 +29,7 @@
     "commitizen": "^3.0.5",
     "cz-conventional-changelog": "^2.1.0",
     "ajv": "^6.10.2",
-    "husky": "^3.0.2",
+    "husky": "1.3.1",
```

Comment:
@sfsholden I had to downgrade this to an older version because newer versions of husky fail to install when CircleCI install git version 2.11.0. I tried installing a newer version of git while building but that didn't work either. Do you have any suggestions on other things we could try ?